vcd_cli/catalog.py

```diff
@@ -114,6 +114,27 @@ def info(ctx, catalog_name, item_name):
     except Exception as e:
         stderr(e, ctx)
 
+
+@catalog.command(short_help='catalog control access details')
+@click.pass_context
+@click.argument('catalog-name',
+                metavar='<catalog-name>',
+                required=True)
+def control_access(ctx, catalog_name):
+    try:
+        client = ctx.obj['client']
+        org_name = ctx.obj['profiles'].get('org')
+        in_use_org_href = ctx.obj['profiles'].get('org_href')
+        org = Org(client, in_use_org_href, org_name == 'System')
+        control_access = org.get_catalog_access_control_settings(catalog_name)
+        stdout('Access Settings for catalog :' + catalog_name)
```

Comment:
Can you post the output of this command?

Reply:
(vcd-cli) malakars-m01:vcd-cli malakars$ vcd catalog control_access cat1
InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised.
Access Settings for catalog :cat1
access_level    href                                                                        name
--------------  --------------------------------------------------------------------------  ----------
FullControl     https://127.0.0.1:8443/api/admin/user/5d088ec4-aa6f-48ec-bb07-6bc756203bbd  org-admin
FullControl     https://127.0.0.1:8443/api/admin/user/8211050a-1aa7-46dc-8974-fdd6d0553ebd  vappauthor
ReadOnly        https://127.0.0.1:8443/api/admin/user/9c0c1d76-e410-408d-a22f-240f4074c02c  vappuser
property            value
------------------  -------
IsSharedToEveryone  false

Reply:
![screen shot 2017-11-21 at 12 18 35 pm](https://user-images.githubusercontent.com/6455111/33094793-342e262c-ceb6-11e7-973b-b1fec63777d9.png)

Reply:
I feel having the order as user_name, user_href, user_access_level is more intuitive. Also making the isSharedToEveryone above the table (blank line between them) will be better.

Reply:
^+1

Reply:
https://jira-hzn.eng.vmware.com/browse/VCDA-283 Add this task to track this.